modules/gatsby/src/templates/doc-page-mdx.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+// import { MDXRenderer } from 'gatsby-mdx';
+
+// Query for the markdown doc by slug
+// (Note: We could just search the allMarkdown from WebsiteConfig ourselves)
+export const query = graphql`
+  query mdxDocById($id: String) {
+    mdx(id: { eq: $id }) {
+      code {
+        body
+      }
+    }
+  }
+`;
+
+// function replaceLinks(props) {
```

Comment:
FYI I'm considering extending this functionality to more elements for the "regular" doc file (ie replacing h1, h2, pre, table etc. DOM elements generated by markdown and whose styling depends on scss by styled components).